src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -136,7 +136,7 @@ public ReadPileup getPileupForLane(final String laneID) {
 
     /**
      * Make a new pileup from elements whose reads belongs to a concrete sample
-     * {@code null} sample retrieves reads without read groups
+     * {@code null} sample retrieves reads without read groups and sample name
```

Comment:
please clarify this to "Passing null sample as an argument retrieves reads whose read group or sample name is null"